openapi/Swarm.yaml

```diff
@@ -0,0 +1,214 @@
+        - in: path
+          name: addr
+          schema:
+            $ref: 'SwarmCommon.yaml#/components/schemas/BzzAddress'
+          required: true
+          description: Bzz address of chunk   
+      responses:
+        '200':
+          description: Retrieved chunk content
+          content:
+            application/octet-stream:
+              schema:
+                type: string  
+                format: binary
+        '400':
+          $ref: 'SwarmCommon.yaml#/components/responses/400'
+        '404':
+          $ref: 'SwarmCommon.yaml#/components/responses/404'
+        '500':
+          $ref: 'SwarmCommon.yaml#/components/responses/500'
+        default:
+          description: Default response
+    post:
+      summary: 'Upload Chunk'
+      tags: 
+        - 'Endpoints on local swarm node'
+      parameters:
+        - in: header
+          name: swarm-tag-uid
+          schema:
+            $ref: 'SwarmCommon.yaml#/components/schemas/UUid'
+          required: false
+          description: UUid of chunk
+        - in: header
+          name: swarm-pin
+          schema:
+            type: boolean
+          required: false
+          description: Represents the pinning state of the chunk
+        - in: path
+          name: addr
+          schema:
+            $ref: 'SwarmCommon.yaml#/components/schemas/BzzAddress'
+          required: true
+          description: Bzz address of chunk   
+      requestBody:
+        content:
+          application/octet-stream:
+            schema:
+              type: string
+              format: binary
+      responses:
+        '200':
+          description: Ok
+          content:
+            application/json:
+              schema:
+                $ref: 'SwarmCommon.yaml#/components/schemas/Hash'
+        '400':
+          $ref: 'SwarmCommon.yaml#/components/responses/400'
+        '500':
+          $ref: 'SwarmCommon.yaml#/components/responses/500'
+        default:
+          description: Default response
+
+  '/files':
+    post:
+      summary: 'Upload data'
+      tags: 
+        - 'Endpoints on local swarm node'
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              properties:
+                file:
+                  type: array
+                  items:
+                    type: string
+                    format: binary
+          file:
+            schema:
+              type: string
+              format: binary
+          
+      responses:
+        '200':
+          description: Ok
+          content:
+            application/json:
+              schema:
+                $ref: 'SwarmCommon.yaml#/components/schemas/Reference'
+        '400':
+          $ref: 'SwarmCommon.yaml#/components/responses/400'
+        '500':
+          $ref: 'SwarmCommon.yaml#/components/responses/500'
+        default:
+          description: Default response
+
+  '/files/{address}':
+    get:
+      summary: 'Get addressed data'
```

Comment:
Same. All data is addressed. I would say: get a file by it's root hash or get a file by it's address